include/orng.hpp

```diff
@@ -307,6 +317,9 @@ CONTRACT orng : public eosio::contract
         uint64_t nonce;
         uint64_t assoc_id;
         bool free_call = false;
+        uint8_t status = 0;   // 0=pending, 1=sent, 2=dead
```

Comment:
maybe we should have REQ_PENDING, REQ_SENT, REQ_DEAD consts to make these self documenting